snaps/how-to/develop-a-snap.md

```diff
@@ -105,13 +105,11 @@ can always build, publish, and use snaps without MetaMask's permission.
 
 ## Resources and tools
 
-You can review the growing number of
-[example snaps](https://github.com/MetaMask/snaps-monorepo/tree/main/packages/examples) maintained
```

Comment:
Why are we removing this?

Reply:
Are we still maintaining it? If so, we can keep it. Sorry!

Reply:
I redid all the examples a couple of months ago and added a lot of documentation etc., and intend to keep this up to date. We also use them for e2e testing, so all features we want to test will have an example snap, so this seems like a good place to point people to.